wallet/src/test/java/io/neow3j/wallet/Nep5Test.java

```diff
@@ -0,0 +1,52 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.http.HttpService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+public class Nep5Test {
```

Comment:
You should not only test the "happy path", but also exceptions or things that can go wrong. Please include tests for that.